meetups/20180502.org

```diff
@@ -0,0 +1,130 @@
+#+TITLE: Math in Emacs: The Basics (and Not-so-basics) of Calc
+#+DATE: 2 May 2018
+#+AUTHOR: with special guest Harry Schwartz
+
+* Tech Check
+*** Wi-Fi
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+
+- SSID "Triplebyte Guest"
+- Password "welcome01"
+* Introduction
+*** Call for Engagement
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+- More venues are always good
+- If you're an "event organizer" you can create any meeting you like
+  - Emacs and Clojure?
+  - Meetings in South/East/North Bay?
+  - More convenient times?
+*** Github Repo
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+https://github.com/Emacs-SF/meetup-notes
+*** Future Topics
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+- Specific guest speakers?
+  - I've run out for now
+  - Maybe one of us has a full-length talk in them?
+- Programming Languages: Python, Clojure, C++
+- Elisp programming?
+- Minor modes?
+- Please LMK or post ideas in group
+*** org-reveal
+#+NAME: Setup
+#+BEGIN_SRC emacs-lisp
+(use-package ox-reveal
+  :config
+  (setq org-reveal-root "https://cdn.jsdelivr.net/npm/reveal.js@3.6.0")
+)
+#+END_SRC
+
+Export with C-c C-e R R, then load file in browser
+* Sponsor
+*** Many thanks to
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+[[file:./triplebyte_logo.png]]
+* Presenter
+*** Harry Schwartz
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+
```

Comment:
I've checked in three scans of the speaker notes that you could update your document to link-to.
look in this directory: https://github.com/Emacs-SF/meetup-notes/tree/master/images/20180502